nodejs/week1/avg.js

```diff
@@ -0,0 +1,9 @@
+const myNumbers = process.argv.slice(2);
```

Comment:
Good idea to extract this confusing value `process.argv` into a variable with an explaining name! ⚡